pkg/types/config.go

```diff
@@ -261,6 +264,36 @@ func (i *InstallSpec) Sanitize() error {
 		}
 	}
 
+	// Setup disk encryption
+	if i.EncryptPersistent {
```

Comment:
Shouldn´t error out if EncryptPersistent is set to true and no secret is provided?

Reply:
> Shouldn´t error out if EncryptPersistent is set to true and no secret is provided?

I actually didn't test this edge-case.. I would expect an empty keyslot to allow anything to unlock it, but I will give it a try!